### PR TITLE
For our monthly issue metrics, Include the date range in the title and label "metrics"

### PR DESCRIPTION
**Pull request by jhkennedy**
Makes it so we can search for all the reports easily:
https://github.com/nsidc/earthaccess/issues?q=is%3Aissue+label%3Ametrics+

And that the specific date range of the report is clear.

<details><summary>Pull Request (PR) draft checklist - click to expand</summary>

- [x] Please review our
      [contributing documentation](https://earthaccess.readthedocs.io/en/latest/contributing/)
      before getting started.
- [x] Populate a descriptive title. For example, instead of "Updated README.md", use a
      title such as "Add testing details to the contributor section of the README".
      Example PRs: [#763](https://github.com/nsidc/earthaccess/pull/763)
- [x] ~~Populate the body of the pull request with:~~
    - A clear description of the change you are proposing.
    - Links to any issues resolved by this PR with text in the PR description, for
      example `closes #1`. See
      [GitHub docs - Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
- [X] ~~Update `CHANGELOG.md` with details about your change in a section titled
      `## Unreleased`.~~ If such a section does not exist, please create one. Follow
      [Common Changelog](https://common-changelog.org/) for your additions.
      Example PRs: [#763](https://github.com/nsidc/earthaccess/pull/763)
- [x] ~~Update the documentation and/or the `README.md` with details of changes to the
      earthaccess interface, if any.~~ Consider new environment variables, function names,
      decorators, etc.

Click the "Ready for review" button at the bottom of the "Conversation" tab in GitHub
once these requirements are fulfilled. Don't worry if you see any test failures in
GitHub at this point!

</details>

<details><summary>Pull Request (PR) merge checklist - click to expand</summary>

Please do your best to complete these requirements! If you need help with any of these
requirements, you can ping the `@nsidc/earthaccess-support` team in a comment and we
will help you out!

- [ ] Add unit tests for any new features.
- [ ] Apply formatting and linting autofixes. You can add a GitHub comment in this Pull
      Request containing "pre-commit.ci autofix" to automate this.
- [ ] Ensure all automated PR checks (seen at the bottom of the "conversation" tab) pass.
- [ ] Get at least one approving review.

</details>

----
📚 Documentation preview 📚: https://earthaccess--841.org.readthedocs.build/en/841/

### CODE REVIEW

**Comment by jhkennedy** [comment]
Question: Should we also close the metrics issues after X time?

**Comment by mfisher87** [comment]
I don't see a reason for metrics issues to have an open status. I think we'll get notified when they're created either way? We could even mark them as "not planned" instead of "closed" to keep them out of the "business as usual" views of issues.

**Review by mfisher87 — APPROVED** [review]
Nice :)

**Comment by jhkennedy** [comment]
> I don't see a reason for metrics issues to have an open status. I think we'll get notified when they're created either way? We could even mark them as "not planned" instead of "closed" to keep them out of the "business as usual" views of issues.

@mfisher87 yep, anyone watching issues should get notified on create. 

I've updated the action to close it right after it's created as "not planned". 

(I wonder if we should sleep a little between open and close?)

**Comment by jhkennedy** [comment]
@mfisher87 would you say this satisfies https://github.com/nsidc/earthaccess/issues/588?

File: .github/workflows/issue-metrics.yml [hunk 1]
```diff
@@ -35,9 +35,15 @@ jobs:
...
+        id: create_issue
         uses: peter-evans/create-issue-from-file@v5
         with:
           title: "Monthly issue metrics report: ${{ env.last_month }}"
           labels: metrics
           token: ${{ secrets.GITHUB_TOKEN }}
           content-filepath: ./issue_metrics.md
+
+      - name: Close Issue
+        run: gh issue close "${{steps.create_issue.outputs.issue-number}}" --reason "not planned"
```
Comment (mfisher87) [inline .github/workflows/issue-metrics.yml:47]: It's that easy? Nice!

**Review by mfisher87 — APPROVED** [review]
Yeah, I think closing as not-planned addresses the original need. :rocket:

File: .github/workflows/issue-metrics.yml (hunk 1 above)
Comment (jhkennedy) [reply .github/workflows/issue-metrics.yml:47]: yep! It's neat.

**Pull request merged**